src/pages/Add-items/Add-items.js

```diff
@@ -95,12 +100,12 @@ export const AddItems = () => {
           <div>
             <label htmlFor="soon">
               <input
-                defaultChecked
                 id="soon"
                 type="radio"
                 name="howSoon"
                 required
                 value={7}
+                checked={product.howSoon === '7'}
```

Comment:
I was wondering the same thing since working on the Issue where these radio buttons were created but finally found why: the `value` property on an `<input>` element is always a string, no matter what type of `input` it is. So even if `value={7}` is taking a number, it is being coerced and returns a string. Anyway I turned them into strings to avoid confusion.